portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Hua Lun's Gallery of Memorable Food</h1>
+
+      <p>I love taking pictures of food.</p>
+      <p>For me, food pics are a story.</p>
+      <p>Overseas trips, school, just life in general.</p>
+      <p>I feel that what I eat represents my mood during that time.</p>
+      <p>It brings back memories and experiences. The journey, the process of obtaining the food.</p>
+      <p>Food just gives me the feels.</p>
+
+      <h3>The Unfortunate Rainbow Sherbet</h3>
+      <img src="images/icecream.jpg" style="width:400px;height:600px;">
```

Comment:
Try to change these to density independent pixels (dp, dip). That will make sure they are compatible with any screen size.